.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Description and Context
+
+<!-- Provide a brief description of the changes introduced by this pull request and the context or motivation behind them. -->
+
+### Tickets ###
+<!-- List any related Jira tickets or GitHub issues -->
+<!-- List any related ADRs or RFCs -->
+
+- [DFC-XXX](https://govukverify.atlassian.net/browse/DFC-XXX)
+
+### Steps to reproduce ###
+<!-- Provide specific instructions for reproducing the changes, if applicable -->
+
+## Checklist
+
+- [ ] **Code Changes**
+  - [ ] Tests added/updated
+  
+- [ ] **Dependencies**
+  - [ ] Dependency versions updated, if necessary
+  
+- [ ] **Testing**
+  - [ ] Local testing done
+  - [ ] Tests run for affected packages
+  
+- [ ] **Documentation**
+  - [ ] Documentation updated, if applicable
```

Comment:
Can we specify the README, tickets and confluence documentation as separate bullets underneath Documentation please? Just to make sure nothing slips through